projects_test.go

```diff
@@ -280,13 +280,15 @@ func TestListOwnedProjects(t *testing.T) {
 func TestEditProject(t *testing.T) {
 	mux, client := setup(t)
 
-	var developerAccessLevel AccessControlValue = "developer"
+	var developerRole AccessControlValue = "developer"
+	developerPipelineVariablesRole := CIPipelineVariables_DeveloperRole
```

Comment:
Goes together with the other comment:

```suggestion
	developerPipelineVariablesRole := CIPipelineVariablesDeveloperRole
```